CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contribute to the urunc project
+
+Urunc is an open-source project licenced under the [Apache License 2.0](LICENSE).
+We welcome anyone who would be interested in contributing to our small project.
+As a first step, please take a look at the following document.
+The current document provides a high level overview of urunc's code structure, along with a few guidelines regarding contributions to the project.
+
+## Table of contents:
+
+1. [Code organization](CONTRIBUTING.md#Code-organization)
+2. [How to contribute](CONTRIBUTING.md#How-to-contribute)
+3. [Opening a new issue](CONTRIBUTING.md#Opening-a-new-issue)
+4. [Requesting a new feature](CONTRIBUTING.md#Requesting-a-new-feature)
+5. [Submitting a PR](CONTRIBUTING.md#Submitting-a-PR)
+6. [Style guide](CONTRIBUTING.md#Style-guide)
+7. [Contact](CONTRIBUTING.md#Contact)
+
+## Code organization
+
+Urunc is written in Go and it has a very small codebase.
```

Comment:
I would leave the small codebase out.

Reply:
Not a negative thing though, but addressed.